src/App.tsx

```diff
@@ -1,61 +1,141 @@
+import React, { useState } from 'react';
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './components/TodoInfo';
+import { User } from './components/UserInfo';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
```

Comment:
1. Use decomposition as much as possible 
2. You can write it with one line (optional, your variant is not bad actually)
```suggestion
  return usersFromServer.find(({ id })=> id === userId) || null;
```